src/geneweaver/api/schemas/apimodels.py

```diff
@@ -0,0 +1,33 @@
+"""Models for API requests."""
+from typing import Iterable, Optional
+
+from fastapi import Query
+from geneweaver.core.enum import GeneIdentifier, Species
+from pydantic import BaseModel
+
+gene_id_type_options = [f"{choice.name} ({choice.value})" for choice in GeneIdentifier]
+species_type_options = [f"{choice.name} ({choice.value})" for choice in Species]
+
+
+class GeneIdMappingResp(BaseModel):
+    """Model for gene id mapping."""
+
+    gene_ids_map: list[dict]
+
+
+class GeneIdMappingReq(BaseModel):
+    """Model for gene id mapping request."""
+
+    source_ids: Iterable[str]
+    target_gene_id_type: GeneIdentifier = Query(
+        None, description=f"Options: {gene_id_type_options}"
```

Comment:
I think you need to do: `Query(..., description="")` to make this show up as required.